ChainReactionBindings/Bindings/SearchPageBindings.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ChainReactionBindings.TestBase;
+using ChainReactionBindings.TestBase.Pages;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow;
+using TechTalk.SpecRun.SpecFlowPlugin.Generator;
+
+namespace ChainReactionBindings.Bindings
+{
+	[Binding]
+	class SearchPageBindings
+	{
+		SearchPage page = new SearchPage();
+
+		[Then(@"I see the search page is loaded")]
+		public void SearchPageLoaded()
+		{
+			if (!page.SearchPageLoaded())
+			{
+				throw new Exception("Search page is not loaded!");
+			}
+		}
+
+		[Then(@"I see the empty search page is loaded")]
+		public void EmptySearchPageLoaded()
+		{
+			if (!page.EmptySearchPageLoaded())
+			{
+				throw new Exception("Empty search page is not loaded!");
+			}
+		}
+
+		[Then(@"I see ""(.*)"" message on search page")]
+		public void ThenISeeMessageOnSearchPage(string text)
+		{
+			switch (text)

```

Comment:
Switch is not an option here, just use parameters and form them to expected message, then compare with actual message

Reply:
done